reports/intro.tex

```diff
@@ -2,15 +2,49 @@
 
 \section{Introduction}
 
-\colorbox{yellow}{RS, TH: A short intro to the protocol}
-
-\paragraph{}\colorbox{yellow}{TODO:}
-
-We have started this project with the Python specification of accountable
-safety in the 3SF Protocol\footnote{URL to the Python specification:
-\url{https://github.com/freespek/ssf-mc/3sf.txt}}. The main goal of the project
-was to demonstrate accountable safety of this protocol by the means of model
-checking.
+At the time of writing this report, Ethereum is using
+Gasper~\cite{buterin2020combining} as the underlying consensus protocol.
+In Gasper, time is divided into slots, which represent intervals during which a new block is proposed to extend the blockchain and undergoes voting. 
+Finalizing a block--ensuring that it is permanently added to the blockchain and cannot be reversed--typically requires 64 to 95 slots.
+This delay in finality makes the network more vulnerable to potential
+block reorganizations when the network conditions change, 
+e.g., during periods of asynchronous network conditions.
+In particular, this finalization delay heightens the network’s exposure to
+Maximal Extractable Value (MEV) exploits, 
+which could undermine the network’s integrity.
+Additionally, the extended finalization period forces users to weigh the
+trade-off between economic security and transaction speed.
+
+To address these issues and speed up finality, D’Amato et al.~\cite{d20243} have recently introduced the \emph{3-slot-finality} (3SF) protocols 
+for Ethereum that achieve finality within three slots after a proposal, hence realizing 3-slot finality.
+This feature is particularly beneficial in practical scenarios where periods of synchrony and robust honest participation often 
+last much longer than the time needed for finalization in the 3SF protocol.
+Finally, the 3SF protocol enhances the practicality of large-scale blockchain networks by enabling the dynamically-available component 
```

Comment:
we have yet to define what the dynamically-available component is

Reply:
I have added a brief description for this